.github/workflows/regression-tests.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           pip install -r dragonfly/requirements.txt
           export DRAGONFLY_PATH="${GITHUB_WORKSPACE}/build/dragonfly" # used by PyTests
 
-          pytest -xvr dragonfly --ignore=dragonfly/replication_test.py
+          pytest -xvrs dragonfly --ignore=dragonfly/replication_test.py
```

Comment:
when s flag is after the r flag this means that the summery printed will show only skipped tests
https://docs.pytest.org/en/6.2.x/usage.html#detailed-summary-report

Reply:
:-o Moved to the beginning and verified locally